WalletWasabi.Fluent/Styles/Themes/BaseDark.axaml

```diff
@@ -2,15 +2,19 @@
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
   <Styles.Resources>
     <Color x:Key="SystemAccentColor">#78A827</Color>
+    <Color x:Key="SystemAccentColorLight1">#719E26</Color>
+    <Color x:Key="SystemAccentColorDark1">#608522</Color>
+    <Color x:Key="SystemControlDisabledBaseLowBrush">#80A3A3A3</Color>
```

Comment:
These are all the same in light and dark themes. Can you move them into the `Base.axaml`?